zcl_text2tab_parser.clas.abap

```diff
@@ -157,11 +159,16 @@ private section.
       !CODE type CHAR2 optional
     raising
       ZCX_TEXT2TAB_ERROR .
-  class-methods BREAK_TO_LINES
+  methods BREAK_TO_LINES
```

Comment:
Let's keep static. Better for unit testing and less bug-prone, just pass comment char as a param